src/Microsoft.DotNet.Wpf/src/WindowsBase/MS/Internal/IO/Packaging/CompoundFile/RightsManagementEncryptionTransform.cs

```diff
@@ -1138,16 +1138,12 @@ private static void
             bool validEnum = false;
 
             // user names: case-insensitive comparison
-            if (((IEqualityComparer) CU.StringCaseInsensitiveComparer).Equals(
-                    authenticationTypeString,
-                    Enum.GetName(typeof(AuthenticationType), AuthenticationType.Windows)))
+            if (string.Equals(authenticationTypeString, nameof(AuthenticationType.Windows), StringComparison.OrdinalIgnoreCase))
```

Comment:
I replaced the usage of `Enum.GetName` in this file with `nameof` since we already know the enum value at build time so `Enum.GetName` is unnecessary overhead.

https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEUCuA7AHwAEAmABgFgAoUgRmuqNoE4AKAUTxwFsA6AcRgYAcgENuMVhgCeABxgQAZqwCCODAAsYeDAEswovRDwAVOTACUaAARrN2vQaOnzvAOq68AEwgB3AM4WFgDcjCyseOIKynZaOvqGusZm8u6ePgFBoVTUAPT51gA8/jzcolDSAHx5BZw8MFCJxtaahtZeMP5gULrAndYAbuVJOP626nGOTXgt5v42YDhQsDoANtLWxuvWAAqi/v6y0BjWot7WHt5+Y+Uw1iWyR1AYMF68NblFuSXcZRXVNAAzNZtDxxvZ4k4ki55NYQNZPBhqABvajWdEY9H5T7FUrlKpozEY7EXdLXU4TBwJZzWUavBEzSBQJ6GO4AEQgZU8ILwA10UGMEh0/l41kJROsJMK3zx/3FRMuGXmDCoEqxBVxv3xALVkoKewOTxOokpkOmCzO1j6tP89Ig6n8ug6m0ULS0pKuAWsHK5M20fIFeCFGBFYtVaqlMq1cvDEoNh2OaBVuqlPz+BNjRJJiuu8aNFIhU2cFpm1rpXgZ9zA2mGEDGvjibruJsL1Ohs1hSib1kgeB+DTDKYKun8eAA5CcANZ4PwzaAI7hGs4nUUAJRgAHNyl5Vp0xl2G0ItFAEScR7t9gnnpsTzmAjYWxp575l2MMBAeaJgLue8Z+yf30HCMCg6bo5BOex2ggRZg1FCV5UxSM021BCMTvfw80TMNUPVHFkJjIdPgASR0BpIlWAtJjbZoZFhAxSzucsWg/J0qUUDY6SgMZdEXVZ9F0DBtl8ASn3UawiDIJt+WsIi2RFHC9U+ABhJYVkEjZNHPCBZBpLYNgeI0xgAIhUPBpGMGAjOseQuOMURRQAZQ/VoTk4CsAFVbWsAAZfRtC84TVgo60N0aUiKwUklALOcy8DuBsP1gABHHBOhDTY4psYAxIE6xAuC5sMAwUQwC0CtAMg5LUv8PQ8A3G0GjgyKNSjdMdTVEiXigcjqAAXyAA=